### DISCUSSION TITLE
max-w-full

### DISCUSSION THREAD

**Discussion by graememullins** [filamentphp/filament#14960] [Help]
### Package

Panel builder

### Package Version

v3

### How can we help you?

Hi,

I'm trying to make the content area, max-w-full from max-w-7xl,

Is this the right code to place in config/filament.php

    'layout' => [
        'max_content_width' => 'full',
    ],
    

It doesn't seem to work, is anything else required?

**Comment by dissto** [comment]
I think using the config is a relict from v2. Is that even in v3? 🤔 

[Customizing the maximum content width](https://filamentphp.com/docs/3.x/panels/configuration#customizing-the-maximum-content-width)

**Comment by enessvg** [comment]
```php
->columnSpanFull()
```
Have you tried this?

**Reply by enessvg** [reply]
@graememullins
